concepticondata/concepticon.tsv

```diff
@@ -2672,3 +2672,8 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2671	PENIS SHEATH	Clothing and grooming	Sheath to cover a penis	Person/Thing	
 2672	HEADWATERS	The physical world	The source of a river or stream	Person/Thing	
 2673	BIRD OF PARADISE	Animals	A bird of the family Paradisaeidae.	Person/Thing	
+2674	ORAL CAVITY	The body	Cavity that is limited by the lips in front, the cheeks on the side, the soft and hard palate at the top and the floor of the mouth below.	Person/Thing	
+2675	MAN OR MALE	Kinship	Relating to the male sex of humans.	Property	
```

Comment:
Yes, the longer I think about it, given that we have the broader gloss "male person", I'd suggest to not introduce this concept here, but to assume, as in the other cases, that the original concept lists does not mean "adult male person or of male gender property", but rather "a male person or an adult male", in which case linking to "male person" should handle it without introducing more concept relations.